examples/python-template-sandbox/agentic-sandbox-client/README.md

```diff
@@ -0,0 +1,114 @@
+# Agentic Sandbox Python Client
```

Comment:
So this is duplicated into `egg-info` (I believe).  I'm not very familiar with checking in `egg-info`. Which one is the source of truth?  Can we use a symlink or a script to avoid them drifting?  (Or some other approach?)